src/media/js/compat_filter.js

```diff
@@ -80,11 +75,6 @@ define('compat_filter',
         views.reload();
     });
 
-    // For mobile, set limit to 10.
-    if (actualFormFactor == 'mobile' || actualPlatform == 'firefoxos') {
-        limit = 10;
```

Comment:
Ah, oops. Got confused since this code was in the compat_filter file. Will restore.